Document/0x05i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -268,23 +255,67 @@ Add the following to build.gradle:
 ### Testing for Debugging Code and Verbose Error Logging
 
 #### Overview
+StrictMode is a developer tool able to detect policy violation, e.g. disk or network access.
+It could be implemented in order to check the use of good coding practices such as no slow code or network access on the main thread.
```

Comment:
Please change to "It can be implemented in order to check the usage of good coding practices such as implementing high-performance code or usage of network access on the main thread."